README.md

```diff
@@ -31,7 +31,7 @@ We are not currently accepting translations of the game on our main repository.
 
 ## License
 
-All code for the content repository is licensed under [MIT](https://github.com/space-wizards/space-station-14/blob/master/LICENSE.TXT).
+Almost code for the content repository is licensed under [MIT](https://github.com/space-wizards/space-station-14/blob/master/LICENSE.TXT). Some code is under the [MPL v2.0](https://www.mozilla.org/en-US/MPL/) and must remain open source.
```

Comment:
I think you need to further clarify this. The "Incompatible With Secondary Licenses" clause of the MPL is not that commonly used and it took me around 40 minutes of googling to understand. I would also specify that "files under the MPL are clearly marked with a comment at the start of the file" because "some code is under the MPL" might lead to a bit of confusion.